js/form/src/ColorPicker.svelte

```diff
@@ -22,6 +23,15 @@
 			dispatch("input");
 		}
 	}
+
```

Comment:
no need for event listener methods if they just dispatch - on:blur and on:focus is sufficient.

Reply:
That make sense.